addons/event/views/event_views.xml

```diff
@@ -163,7 +163,8 @@
             <field name="res_model">event.registration</field>
             <field name="name">Attendees</field>
             <field name="view_mode">kanban,tree,form,calendar,graph</field>
-            <field name="context">{'search_default_event_id': active_id, 'default_event_id': active_id, 'search_default_expected': True}</field>
+            <field name="domain">[('event_id', '=', active_id)]</field>
+            <field name="context">{'default_event_id': active_id, 'search_default_expected': True}</field>
```

Comment:
From spec :
> 2) In single event mode (coming from the stat button of an event), add a domain so that only attendees of this event are shown. We hence remove the default filter "Event I'm coming from".

Also, now "event.name" in kanban-card is invisible when "default_event_id" is set. If we do not apply a domain to limit to the "active-id" event, the user can remove the filter and so all attendees are shown without there event name...
This is also the reason why added a kanban view in Reporting=>Attendees...

Any advice?

Reply:
I just missed the spec point this morning, not enough coffee, you can keep as it is :)